src/test/resources/features/CreateMatchResult.feature

```diff
@@ -0,0 +1,25 @@
+Feature: CreateMatchResult
+  In order to use the app
+  As a user
+  I want to see the result of a Match
+
+  Background:
+    Given There is a player "user1" with password "password" and "email@example.com"
+    And There is a match with id 1
+
+  Scenario: Play a Match
+    Given  There is no registered matchResult for this match attached to the player
+    When   The match is finished with a matchResult with result 1 and time 1
```

Comment:
You have added the security matchers allowing only authenticated calls to `/matchResults`. However, you are not doing authenticated calls, which will result in 401 Unauthorized status. You must include the `I login as....` step.